sh/tests/solutions/comparator.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
```

Comment:
Should the script test if it is run with the correct number of arguments?

Reply:
Done